launch_testing/launch_testing/pytest/hooks.py

```diff
@@ -110,8 +110,16 @@ def pytest_pycollect_makemodule(path, parent):
         if module is not None:
             return module
     if path.basename == '__init__.py':
-        return pytest.Package(path, parent)
-    return pytest.Module(path, parent)
+        try:
+            # since pytest 5
```

Comment:
I have currently installed pytest 5.1, and `.from_parent` isn't available.
Based on the [changelog](https://docs.pytest.org/en/latest/changelog.html#deprecations), I would say the change was introduced in `5.4.0` (search `from_parent`).